docs/architecture/index.rst

```diff
@@ -1,25 +1,529 @@
+.. _det-system-architecture:
+
+.. _system-architecture:
+
 #####################
  System Architecture
 #####################
 
-Determined consists of a single **master** and one or more **agents**. There is typically one agent
-per compute server; a single machine can serve as both a master and an agent.
+.. pull-quote::
+
+   Determined makes it easy to transform data into probabilities.
```

Comment:
While I agree with the sentiment, I don't remember saying this, I don't think a quote from me carries much weight with anyone, and generating probabilities is not the only goal of machine learning tasks in general.  I don't think a quote from anyone is needed, but maybe a simple headline statement about how Determined coordinates complex resources behind the scenes, letting Machine Learning Engineers focus on others tasks.

Reply:
So serious.